interface/main/finder/dynamic_finder_ajax.php

```diff
@@ -145,7 +146,13 @@
         } else if ($colname == 'DOB' || $colname == 'regdate' || $colname == 'ad_reviewed' || $colname == 'userdate1') {
             $arow[] = oeFormatShortDate($row[$colname]);
         } else {
-            $arow[] = $row[$colname];
+            $frow = sqlQuery(
+                "SELECT * FROM layout_options " .
+                "WHERE form_id = 'DEM' AND field_id = ? LIMIT 1",
+                array($colname)
+            );
```

Comment:
2 quick questions here:
1. Will it be expensive to do a query for every value, or is there a way to do one query of layout_options into an array and then use that array to search instead(for example the $colname is the key for the array returned)?
2. Can the above else if be removed since generate_plaintext_field should now do this?